grammar_ne.js

```diff
@@ -0,0 +1,114 @@
+// Generated automatically by nearley, version 2.16.0
```

Comment:
This should not be committed (it doesn't need to be committed to go to npm)

Reply:
Good point